PhotoEnhancer/PhotoEnhancer/Filters/NoiseParameters.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class NoiseParameters : IParameters
+    {
+        public double Intensity { get; set; }
+
+        public ParameterInfo[] GetDescription()
+        {
+            return new[]
+            {
+                new ParameterInfo() {
+                    Name = "Интенсивность",
```

Comment:
Название фильтра неудачное. Смотришь на выпадающий список фильтров и не понимаешь, что же этот фильтр делает. Лучше было использовать название из задания.
```suggestion
                    Name = "Шум",
```